results/Analyzing_4_Million_Jupyter_Notebooks.md

```diff
@@ -0,0 +1,92 @@
+# Analyzing 4 Million Jupyter Notebooks
+
+## A follow-up to *Study 1: Analyzing 1 Million Jupyter Notebooks* from [Design and Use of Computational Notebooks](https://adamrule.com/files/dissertation_rule.pdf) by Adam Rule
+
+
+In July of 2017, Adam Rule “scraped and analyzed 1.23 million publicly available Jupyter Notebooks on GitHub” (Rule, 23). In a section of his PhD Dissertation, he reports statistics reflecting the role of narrative in computational notebooks. His report focuses on languages and packages used and the amount of context provided by users in terms of the amount of markdown compared to code and repository descriptions. Here, we inspect how these aspects of the use of Jupyter Notebooks has changed over the past two years. The structure of this paper follows that of section 3.2 of Adam’s dissertation, discussing the results of my analysis and how they compare to his 2017 results. 
+
+The number of notebooks on GitHub has grown dramatically since Rule’s research. Between July 2017 and July 2019, the number of notebooks on GitHub have over tripled, now at 5,376,454 notebooks and 414,614 users. The methods I used to query, download, and extract data from GitHub is based on Adam’s research. His code can be found in Jupyter Notebooks on GitHub under [activityhistory/jupyter_on_github](https://github.com/activityhistory/jupyter_on_github), and my scripts can be found on GitHub under[jupyter-resources/notebook-research](https://github.com/jupyter-resources/notebook-research).
```

Comment:
are

Reply:
"are based on Adam's research" didn't realize this would just comment on the whole block of text lol